src/utils/functionGeneration.ts

```diff
@@ -133,6 +113,97 @@ export function createCairoFunctionStub(
   return funcDef;
 }
 
+export function createImportFuncFuncDefinition(
```

Comment:
FuncFuncy funky one

Reply:
It seems the problem is the node class naming. `cairoImport` node wouldn't cause the problem. We can create an issue for that if that's too much work for now